src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Windows/MyExtensionsApp._1.Windows.csproj

```
@@ -250,4 +250,10 @@
   </PropertyGroup>
 
   <Import Project="..\MyExtensionsApp._1.Shared\base.props" />
+  
+  <Target Name="ValidateOverrides" BeforeTargets="Restore;_CheckForUnsupportedTargetFramework" Condition="'$(OverrideTargetFramework)' != ''">
+    <Error
+        Text="OverrideTargetFramework set to '$(OverrideTargetFramework)' is missing Windows target. Set OverrideTargetFramework to $baseTargetFramework$-windows10.0.19041.0 or skip building this project (eg unload the project in Visual Studio)"
+        Condition="'$(OverrideTargetFramework)'!='$(DotNetVersion)-windows10.0.19041.0'" />
```

```suggestion
        Condition="!$(OverrideTargetFramework.Contains('windows10'))" />
```
We probably want to avoid using a fully qualified check as the developer could change which version of Windows they are targeting and this check would then break.